indexer/reindex_bags_backlog.py

```diff
@@ -0,0 +1,191 @@
+#!/usr/bin/env python3
+"""
+This script will send every bag in the storage service to the bags indexer
+to be re-indexed in Elasticsearch.
+"""
+
+import json
+import uuid
+
+import boto3
+from elasticsearch import Elasticsearch
+from elasticsearch_dsl import Search
+
+
+def scan_table(dynamo_client, *, TableName, **kwargs):
+    paginator = dynamodb_client.get_paginator("scan")
+
+    for page in paginator.paginate(TableName=TableName, **kwargs):
+        yield from page["Items"]
+
+def create_client(service_name, role_arn):
+    def _assumed_role_session(role_arn):
+        sts_client = boto3.client('sts')
+
+        assumed_role_object=sts_client.assume_role(
+            RoleArn=role_arn,
+            RoleSessionName=uuid.uuid1().hex
+        )
+
+        credentials=assumed_role_object['Credentials']
+
+        session = boto3.Session(
+            aws_access_key_id=credentials['AccessKeyId'],
+            aws_secret_access_key=credentials['SecretAccessKey'],
+            aws_session_token=credentials['SessionToken']
+        )
+
+        return session
+
+    session = _assumed_role_session(role_arn)
+    return session.client(service_name)
+
+# The bag indexer only cares about space, externalIdentifier & version
+def fake_known_replicas_payload(space, externalIdentifier, version):
+    return {
+        'context': {
+            'ingestId': "8368b576-8206-4552-870c-005186f0264a",
+            'ingestType': {
+                "id": "create"
+            },
+            'storageSpace': space,
+            'ingestDate': "2069-11-30T03:09:31.986Z",
+            'externalIdentifier': externalIdentifier,
+        },
+        'version': version,
+        'knownReplicas': {
+            'location': {
+                'provider': {
+                    "type": "AmazonS3StorageProvider"
+                },
+                'prefix': {
+                    "namespace": "go3dmZxK",
+                    "path": "XrLuVFbL"
```

Comment:
Maybe worth setting these to explicit strings to make it easier to debug if it ends up in the wrong place?
```suggestion
                    "namespace": f"sent by {__file__}",
                    "path": f"sent by {__file__}"
```